auth/auth.go

```diff
@@ -310,12 +322,23 @@ func (myAuth *Auth) AwsS3Credentials(level string, bucket string, prefix string)
 //   * auth:azure-table-access:<account>/<table>
 //
 // See http://docs.taskcluster.net/auth/api-docs/#azureTableSAS
-func (myAuth *Auth) AzureTableSAS(account string, table string) (*AzureSharedAccessSignatureResponse, *tcclient.CallSummary, error) {
+func (myAuth *Auth) AzureTableSAS(account, table string) (*AzureSharedAccessSignatureResponse, *tcclient.CallSummary, error) {
 	cd := tcclient.ConnectionData(*myAuth)
 	responseObject, callSummary, err := (&cd).APICall(nil, "GET", "/azure/"+url.QueryEscape(account)+"/table/"+url.QueryEscape(table)+"/read-write", new(AzureSharedAccessSignatureResponse), nil)
 	return responseObject.(*AzureSharedAccessSignatureResponse), callSummary, err
 }
 
+// Returns a signed URL for AzureTableSAS, valid for the specified duration.
+//
+// Required scopes:
+//   * auth:azure-table-access:<account>/<table>
+//
+// See AzureTableSAS for more details.
+func (myAuth *Auth) AzureTableSAS_SignedURL(account, table string, duration time.Duration) (*url.URL, error) {
```

Comment:
Nope, not usually. The reason I did this was just to make it easier to see that this is the signed url method for `AzureTableSAS` and I thought it might make easier reading than e.g. `AzureTableSASSignedURL`. Do you think the latter is better? I can't decide myself... :/